#!/usr/bin/env python
PACKAGE = "dynamic_pid"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
gen.add("dKP", double_t, 0, "A double KP",    .67, 0, 2.0)
gen.add("dKI", double_t, 0, "A double KI",    .00, 0, 0.2)
gen.add("dKD", double_t, 0, "A double KD",    .51, 0, 1.0)


gen.add("VdKP", double_t, 0, "A double KP",    .3, 0, 1.0)
gen.add("VdKD", double_t, 0, "A double KI",    .14, 0, 1.0)

exit(gen.generate(PACKAGE, "dynamic_pid", "PIDController"))